source/commands/nick.cpp

```diff
@@ -1,9 +1,38 @@
 #include "commands.hpp"
 
+bool checkNick(std::string nick)
+{
+	if (!((nick[0] >= 'a' && nick[0] <= 'z') || (nick[0] >= 'A' && nick[0] <= 'Z')))
+		return (false);
+	return (true);
+}
```

Comment:
The whole Nickname should be checked. RFC1459 says: `<letter> { <letter> | <number> | <special> }`. We might want to think about adding functions like `is_letter`, `is_number`, `is_nonwhite`, etc. to a common place, so we have to write them only once.

Reply:
```suggestion
bool checkNick(std::string nick)
{
	for (size_t x = 0; x < nick.length(); x ++)
	{
	if (!((nick[x] >= 'a' && nick[x] <= 'z') || (nick[x] >= 'A' && nick[x] <= 'Z')))
		return (false);
	}
	return (true);
}
```